src/Deserializers/LegacySnakDeserializer.php

```
@@ -41,11 +42,7 @@ public function deserialize( $serialization ) {
 				return new PropertySomeValueSnak( $serialization[1] );
 			case 'value':
 				return $this->deserializeValueSnak( $serialization );
-				// @codeCoverageIgnoreStart
-			default:
-				throw new LogicException();
```

-1. The code should fail if that happens and not silently return null.